bindings/ros/aditof_roscpp/src/aditof_utils.cpp

```diff
@@ -11,13 +11,14 @@ std::shared_ptr<Camera> initCameraEthernet(int argc, char **argv) {
     google::InitGoogleLogging(argv[0]);
     FLAGS_alsologtostderr = 1;
     Status status = Status::OK;
-
+    std::string ip;
     if (argc < 2) {
-        LOG(ERROR) << "No ip provided!";
-        return nullptr;
+        LOG(WARNING) << "No ip provided!";
```

Comment:
I think INFO is better suited here. Client gets informed that in absence of an IP, an attempt to connect to the camera through USB will be made. Which means the message should be updated to explain these things.

Reply:
You're right. Fixed in commit https://github.com/analogdevicesinc/aditof_sdk/pull/250/commits/9f03a7a6556856aa50f986fbbbf5b0ceffd0dab8